processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/beq.asm (1 of 24)
processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/test.ldst1.asm (1 of 2)
--- memsim.hex	2022-02-11 04:18:38.140026383 -0500
+++ memcpu.hex	2022-02-11 04:18:38.098024797 -0500
@@ -15,7 +15,7 @@
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:04004000000000F0CC
+:04004100000000F4C7
+:04004200000000F8C2
 :00000001FF
processing file asmFiles/test.ldst2.asm (2 of 2)
--- memsim.hex	2022-02-11 04:18:40.778125997 -0500
+++ memcpu.hex	2022-02-11 04:18:40.717123693 -0500
@@ -10,10 +10,10 @@
 :04000C00AC450008F7
 :04000D00AC4A000CED
 :04000E00FFFFFFFFF2
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
-:04020300FEEDBEEF5F
+:0402000000000F00EB
+:0402010000000F04E6
+:0402020000000F08E1
+:040203000000BEEF4A
 :0403C000000073378F
 :0403C1000000270110
 :0403C20000001337ED
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/try.asm (1 of 1)
